### DISCUSSION TITLE
component proposal

### DISCUSSION THREAD

**Discussion by sayanChaterjee** [tailwindlabs/tailwindcss#14719] [Help]
Hey I am new in this community. I would like to propose contributing a new component. Could you please let me know whether I can add a component or not. Looking forward to your feedback and approval!

**Comment by mary598chess** [comment]
Welcome to the community! It's fantastic that you want to contribute. You should be able to add a new component. I recommend checking the contribution guidelines or reaching out to a community moderator for specific instructions. Looking forward to seeing your contribution! [iadpworkforcenow.com](https://www.iadpworkforcenow.com)
